### ISSUE TITLE
Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by atifwaqar** [StarcounterApps/People#98]
### Steps to reproduce
No idea so far, Probably somewhere creating relations between an organization and a person.

### Technical information
```
"System.NullReferenceException: Object reference not set to an instance of an object.
   at Simplified.Ring1.Something.get_Name()
   at People.PersonsPage.PersonsPersonPage.<>c.<AddParentsToOrganizations>b__9_0(Organization x) in C:\adfenix-codebase\adfenix-backend\People\src\People\ViewModels\PersonsPage.json.cs:line 129
   at System.Linq.EnumerableSorter`2.ComputeKeys(TElement[] elements, Int32 count)
   at System.Linq.EnumerableSorter`1.Sort(TElement[] elements, Int32 count)
   at System.Linq.OrderedEnumerable`1.<GetEnumerator>d__1.MoveNext()
   at Starcounter.Json.Array_InitializeAfterImplicitConversion(Json parent, TObjArr template) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.XSON\Class Json\Json.Array.cs:line 79
   at Starcounter.Advanced.XSON.JsonExtension.Scope(Json json, Action action) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.XSON\Class Json\JsonExtension.cs:line 68
   at People.PersonsPage.PersonsPersonPage.AddParentsToOrganizations() in C:\adfenix-codebase\adfenix-backend\People\src\People\ViewModels\PersonsPage.json.cs:line 129
   at People.PersonsPage.PersonsPersonPage.OnData() in C:\adfenix-codebase\adfenix-backend\People\src\People\ViewModels\PersonsPage.json.cs:line 139
   at Starcounter.Advanced.XSON.JsonExtension.Scope(Json json, Action action) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.XSON\Class Json\JsonExtension.cs:line 68
   at Starcounter.Json.Array_InitializeAfterImplicitConversion(Json parent, TObjArr template) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.XSON\Class Json\Json.Array.cs:line 96
   at Starcounter.Templates.TArray`1.BoundOrUnboundSet(Json parent, Arr`1 value) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.XSON\Templates\TArray.cs:line 116
   at People.PersonsPage.set_Persons(Arr`1 value) in C:\adfenix-codebase\adfenix-backend\People\src\People\obj\x64\Debug\ViewModels\ViewModels\PersonsPage.json:line 20
   at People.PersonsPage.RefreshPersons() in C:\adfenix-codebase\adfenix-backend\People\src\People\ViewModels\PersonsPage.json.cs:line 20
   at People.PartialHandlers.<>c.<Register>b__0_1() in C:\adfenix-codebase\adfenix-backend\People\src\People\Api\PartialHandlers.cs:line 39
   at Starcounter.Rest.UserHandlerInfo.RunUserDelegate(Request req, IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Rest\UriHandlersManager.cs:line 341
   at Starcounter.Internal.Web.AppRestServer.RunDelegateAndProcessResponse(IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack, Request req) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Apps.JsonPatch\AppRestServer.cs:line 98
   at Starcounter.Rest.UriManagedHandlersCodegen.RunUriMatcherAndCallHandler(String methodSpaceUriSpace, String methodSpaceUriSpaceLower, Request req, UInt16 portNumber, Response& resp) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Rest\UriManagedHandlersCodegen.cs:line 1133
   at Starcounter.Self.DoSelfCall(UInt16 portNumber, String method, String relativeUri, Dictionary`2 headersDictionary, String body, Byte[] bodyBytes, HandlerOptions handlerOptions, Request req) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Internal\Rest\Self.cs:line 397
   at Starcounter.Self.GET[T](String uri, HandlerOptions ho) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Internal\Rest\Self.cs:line 22
   at People.MainHandlers.GetLauncherPage(String Url, Boolean DbScope) in C:\adfenix-codebase\adfenix-backend\People\src\People\Api\MainHandlers.cs:line 138
   at People.MainHandlers.<>c.<Register>b__0_5() in C:\adfenix-codebase\adfenix-backend\People\src\People\Api\MainHandlers.cs:line 65
   at People.AuthorizedHandle.<>c__DisplayClass0_0.<GET>b__1() in C:\adfenix-codebase\adfenix-backend\People\src\People\Helpers\AuthorizedHandle.cs:line 9
   at People.AuthorizedHandle.TrySignIn(Func`1 code) in C:\adfenix-codebase\adfenix-backend\People\src\People\Helpers\AuthorizedHandle.cs:line 21
   at People.AuthorizedHandle.<>c__DisplayClass0_0.<GET>b__0() in C:\adfenix-codebase\adfenix-backend\People\src\People\Helpers\AuthorizedHandle.cs:line 9
   at Starcounter.Rest.UserHandlerInfo.RunUserDelegate(Request req, IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Rest\UriHandlersManager.cs:line 341
   at Starcounter.Internal.Web.AppRestServer.RunDelegateAndProcessResponse(IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack, Request req) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Apps.JsonPatch\AppRestServer.cs:line 98
   at Starcounter.Internal.AppsBootstrapper.ProcessExternalRequest(Request req) in C:\TeamCity\TeamCity10\buildAgent\work\sc-3487\Level1\src\Starcounter.Apps.JsonPatch\AppsBootstrapper.cs:line 765
HResult=-2147467261
"
```
##### Starcounter version 
2.3.0.5916

##### Involved apps' version
UserAdmin
SignIn
Launcher
People

**Comment by atifwaqar** [comment]
My guess is that this exception occers somewhere in these function : 
```cs
QueryResultRows<Somebody> Org_Members = OrganizationMember.MembersOfAnOrganization(org);
if (!Org_Members.Any(item => item.Key == sub_org.Key))
{
    UserRelations = new OrganizationMember();
    UserRelations.Organization = org;
    UserRelations.Member = sub_org;
}
```

**Comment by bouguila** [comment]
closing as fixed in StarcounterApps/Images#103
